fluid/text_matching_on_quora/configs/basic_config.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
```

Comment:
you should add copy right license here

Reply:
Added copy right license : https://github.com/PaddlePaddle/models/pull/1367/commits/f421c0785a93b295833109733dc54bbd4c3420aa